### ISSUE TITLE
Rollout Baseline Survey

### ISSUE THREAD

**Issue by productmike** [department-of-veterans-affairs/va.gov-cms#13430]

**Comment by joagnitti** [comment]
This ticket is currently blocked by #13662. The survey is currently in the VIEWS process and will be blocked until we receive approval (the timeline of which is unknown).

**Comment by productmike** [comment]
technically #13501 is blocked by the VIEWS process. moving this back to future sprint. all tickets in this epic #13312 will be on hold until the VIEWS process is completed
